src/main/java/core/basesyntax/ArrayList.java

```diff
@@ -1,53 +1,102 @@
 package core.basesyntax;
 
+import java.util.NoSuchElementException;
+
 /**
  * <p>Реалізувати свій ArrayList який імплементує інтерфейс List. Дотриматися основних вимог щодо
  * реалізації ArrayList (default capacity, newCapacity...)</p>
  */
 public class ArrayList<T> implements List<T> {
 
+    private int defaultCapacity;
```

Comment:
DEFAULT_CAPACITY should be a constant, you can return it the way it was 
i was talking only about variable newCapacity that you deleted